homeassistant/components/knx/climate.py

```diff
@@ -2,7 +2,7 @@
 from typing import List, Optional
 
 from xknx.devices import Climate as XknxClimate
-from xknx.dpt import HVACOperationMode
+from xknx.dpt import HVACControllerMode, HVACOperationMode
```

Comment:
previously we had one attribute for OperationMode and ControllerMode. We changed that to support both simultaneously. ControllerMode is "Mode" in HA terms and OperationMode is "Preset" in HA terms.

this is why we changed some "OPERATION_MODE*" variables to "CONTROLLER_MODE*"